### PR TITLE
Add 'newer alluxio version available' message to WEB UI

### PR DESCRIPTION
**Pull request by tieujason330**
- adds a notification icon to the `master's` web ui if a newer version of Alluxio is available
Screen Shot 2019-08-29 at 17 06 20
Screen Shot 2019-08-29 at 17 06 39

### CODE REVIEW

**Comment by tieujason330** [comment]
@calvinjia @madanadit PTAL, thanks

File: core/common/src/main/java/alluxio/wire/MasterWebUIInit.java [hunk 1]
```diff
@@ -49,9 +49,9 @@ public boolean getDebug() {
   }
 
   /**
-   * Gets latest version.
+   * Gets newer version available.
    *
-   * @return the latest version
+   * @return the newer version available
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: `true if a newer version is available, false otherwise`

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: core/common/src/main/java/alluxio/wire/MasterWebUIInit.java (hunk 1 above)
Comment (tieujason330) [reply core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: updated

Comment (tieujason330) [reply core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: updated

**Comment by madanadit** [comment]
jenkins, test this please.

**Comment by madanadit** [comment]
```
Tests in error: 
alluxio.client.cli.JournalToolTest.dumpHeapCheckpointFromEmbeddedJournal(alluxio.client.cli.JournalToolTest)
  Run 1: JournalToolTest.dumpHeapCheckpointFromEmbeddedJournal:165->checkpointEmbeddedJournal:212->lambda$checkpointEmbeddedJournal$0:226 Runtime
  Run 2: JournalToolTest.dumpHeapCheckpointFromEmbeddedJournal:165->checkpointEmbeddedJournal:212->lambda$checkpointEmbeddedJournal$0:226 Runtime
```

File: core/common/src/main/java/alluxio/wire/MasterWebUIInit.java [hunk 2]
```diff
@@ -103,6 +113,17 @@ public MasterWebUIInit setDebug(boolean debug) {
     return this;
   }
 
+  /**
+   * Sets newer version available.
+   *
+   * @param newerVersionAvailable the newer version available
+   * @return true if a newer version is available, false otherwise
+   */
+  public MasterWebUIInit setNewerVersionAvailable(boolean newerVersionAvailable) {
```
Comment (madanadit) [inline core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: the `@return` javadoc seems to be incorrect

File: core/common/src/main/java/alluxio/wire/MasterWebUIInit.java [hunk 3]
```diff
@@ -117,7 +117,7 @@ public MasterWebUIInit setDebug(boolean debug) {
    * Sets newer version available.
    *
    * @param newerVersionAvailable the newer version available
-   * @return true if a newer version is available, false otherwise
+   * @return the newer version available
```
Comment (madanadit) [inline core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: wait this is still wrong

Comment (madanadit) [reply core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: actually all `set*` methods have an incorrect javadoc in this file

Comment (tieujason330) [reply core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: i was trying to match the other `set` method `@return` comments. what would be better?

Comment (madanadit) [reply core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: you could fix all methods. with `@return the updated MasterWebUIInit instance`

Comment (tieujason330) [reply core/common/src/main/java/alluxio/wire/MasterWebUIInit.java]: updated

**Comment by madanadit** [comment]
alluxio-bot, merge this please.

**Pull request merged**